src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -279,7 +279,7 @@ namespace OpenMS
         }
 
         // Add a new spectrum, initialize and set MS level and RT
-        spectrum_data_.resize(spectrum_data_.size() + 1); // TODO !!
+        spectrum_data_.resize(spectrum_data_.size() + 1);
```

Comment:
this code is really old. I think what this basically does is avoiding the hassle to specify the type of the spectrum and just use the default ctor. Its doable I guess, but not the most explicit solution.